### ISSUE TITLE
[feature request] local file search with multiple directories

### ISSUE THREAD

**Issue by SUCM** [jacquesh/foo_openlyrics#193]
Hi,

is it possible to specify multiple directories for local file search of existing lyric file? for example, I use below pattern to specify 2 different directories, separated by **||**, to look for lyrics with [Lyric Show Modoki](https://ashiato1.blog.fc2.com/blog-entry-64.html)

$directory_path(%path%)\lyrics\$replace(%title%,\,,/,,*,,?,,",,<,,>,,|,,:,)||$directory_path(%path%)\..\lyrics\$replace(%title%,\,,/,,*,,?,,",,<,,>,,|,,:,)

**Comment by jacquesh** [comment]
Potentially, although it seems unlikely to be worthwhile. What about your setup prevents you from using a single directory?

**Comment by SUCM** [comment]
on my setup, I typically have structures like below, so there are 2 possible directory structures to look for lyrics based on where the song is. the other thing is that I want to make sure OpenLyric will never overwrite the lyrics file (due to bugs or found some online lyrics) that I already adjusted in the album directory

for single disc albums
```
album_directory/lyrics/song1.lrc
album_directory/lyrics/song2.lrc
album_directory/lyrics/song3.lrc
album_directory/song1.flac
album_directory/song2.flac
album_directory/song3.flac
```
for album with more than 1 discs
```
album_directory/lyrics/song1.lrc
album_directory/lyrics/song2.lrc
album_directory/lyrics/song3.lrc
album_directory/disc1/song1.flac
album_directory/disc1/song2.flac
album_directory/disc1/song3.flac
album_directory/disc2/song4.flac
album_directory/disc2/song5.flac
album_directory/disc2/song6.flac
album_directory/disc3/song7.flac
album_directory/disc3/song8.flac
album_directory/disc3/song9.flac
```

**Comment by jacquesh** [comment]
Fair enough, that seems reasonable. The functionality itself should be fairly simple, but the UI for configuring it will probably be rather more involved.

**Comment by SUCM** [comment]
I remember the old lyric3 would let you double click on the Local File Search to pop up a configuration panel for it. perhaps you can have a panel similar to the searching order but takes 1 directory pattern per line (or item) for Local File Search
